projects/iio_demo/src/app/main.c

```diff
@@ -71,9 +77,58 @@ static uint8_t out_buff[MAX_SIZE_BASE_ADDR];
 
 #endif
 
+#ifdef STM32_PLATFORM
+void Error_Handler(void)
+{
+	/* User can add his own implementation to report the HAL error return state */
+	__disable_irq();
+	while (1) {
+	}
+}
+
+void SystemClock_Config(void)
```

Comment:
Will this be the same for all project using stm32?
Can this be added to drivers/platform/stm32/platform_init.c like https://github.com/analogdevicesinc/no-OS/pull/880/commits/31145ea6d4d531a57df53f1270d1f78935838458 ?

Reply:
kind of, yes
HAL_Init and SystemClock_Config will have to be called in each project but they are both autogenerated (by STM32CubeIDE) and custom for each project, the user needs to copy them anyway to their project and I think it's useless to create another function (say platform_init) that calls these two functions. Also, some users may initialize more stuff than these two functions do, so better keep things simple I guess